web/app/js/components/AddResources.jsx

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import { incompleteMeshMessage } from './util/CopyUtils.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class AddResources extends React.Component {
+  static propTypes = {
+    resourceName: PropTypes.string.isRequired,
+    resourceType: PropTypes.string.isRequired
+  }
+
+  render() {
+    const {resourceName, resourceType} = this.props;
+
+    return (
+      <div className="mesh-completion-message">
+        {_.startCase(resourceType)} {resourceName} is not in the mesh.
```

Comment:
I think we need to use `friendlyTitle` here instead of `_.startCase`:

![image](https://user-images.githubusercontent.com/9226/44928921-18155a00-ad0e-11e8-90e2-718ba1192f0e.png)